x-pack/plugins/triggers_actions_ui/public/application/sections/rule_form/rule_form.tsx

```diff
@@ -660,14 +660,22 @@ export const RuleForm = ({
       : false;
 
     if (MULTI_CONSUMER_RULE_TYPE_IDS.includes(rule?.ruleTypeId ?? '')) {
+      // Use selectedConsumer when creating a new rule, existing rule consumer when editing
+      const ruleConsumer = initialSelectedConsumer ? selectedConsumer : rule.consumer;
```

Comment:
might be better to check the `operation` prop? might not be apparent that initialSelectedConsumer is only defined during editing

Reply:
`operation` prop is apparently defined by `i18n` translations. This is baffling to me, it seems like this would break any code that relies on it when the user's language isn't English.

So anyway I agree with you on principle, but in practice maybe we shouldn't